### PR TITLE
fixed cob_voltage_controls topic names

### PR DESCRIPTION
**Pull request by benmaidel**
fixes:
- renamed and remaped voltage and em state topic names
- removed console debugging output

this PR is coupled with ipa320/cob_robots#360

### CODE REVIEW

File: cob_voltage_control/common/src/cob_voltage_control_common.cpp [hunk 1]
```diff
@@ -28,9 +28,9 @@ class cob_voltage_control_data
     int in_phidget_current;
 
     //output data
-    cob_msgs::PowerBoardState out_pub_em_stop_state_;
+    cob_msgs::PowerBoardState out_pub_relayboard_state;
```
Comment (fmessmer) [inline cob_voltage_control/common/src/cob_voltage_control_common.cpp]: Should we call it `out_pub_powerboard_state` to be consistent with the message type and to remove any (obsolete/missleading) relation to relayboard?

File: cob_voltage_control/ros/src/cob_voltage_control_ros.cpp [hunk 2]
```diff
@@ -50,14 +50,14 @@ class cob_voltage_control_ros
 
         cob_voltage_control_ros()
         {
-            topicPub_powerstate = n_.advertise<cob_msgs::PowerBoardState>("pub_em_stop_state_", 1);
-            topicPub_em_stop_state_ = n_.advertise<cob_msgs::EmergencyStopState>("pub_relayboard_state_", 1);
+            topicPub_powerstate = n_.advertise<cob_msgs::PowerBoardState>("relayboard_state", 1);
```
Comment (fmessmer) [inline cob_voltage_control/ros/src/cob_voltage_control_ros.cpp]: Maybe also change the topic name to `powerboard_state`?

**Comment by fmessmer** [comment]
It seems using "relayboard" has been very confusing a number of times...how about completely replacing "relayboard" by respective "powerboard"?

**Comment by benmaidel** [comment]
yes, we can do this.

**Comment by benmaidel** [comment]
done

**Comment by benmaidel** [comment]
tested on raw3-3!

**Comment by floweisshardt** [comment]
LGTM

**Comment by NadiaHG** [comment]
LGTM

**Comment by fmessmer** [comment]
Formal Review OK

**Pull request merged**